nixos/modules/security/pam.nix

```diff
@@ -253,6 +267,8 @@ let
               "auth sufficient ${pkgs.pam_u2f}/lib/security/pam_u2f.so"}
           ${optionalString cfg.usbAuth
               "auth sufficient ${pkgs.pam_usb}/lib/security/pam_usb.so"}
+          ${optionalString cfg.googleAuth
+              "auth required ${pkgs.googleAuthenticator}/lib/security/pam_google_authenticator.so"}
```

Comment:
That is to say, I'm deliberately not allowing login with *just* OTP. There aren't enough combinations. It'd take a while, but with the default rate-limit you could guess a valid OTP in a couple of... weeks, I think, at most.